### PR TITLE
Drop obsolete code and seal classes/methods

### PR DESCRIPTION
**Pull request by Corniel**
Over the lifetime of version 6, multiple classes/methods have been decorated as obsolete, or as being sealed in the next major version. This PR processes them all.

### CODE REVIEW

File: src/Qowaiv/Hashing/Hash.cs [hunk 1]
```diff
@@ -31,7 +31,6 @@
     [Pure]
     [EditorBrowsable(EditorBrowsableState.Never)]
     [DoesNotReturn]
-    [WillBeSealed]
     public override int GetHashCode() => NotSupportedBy<Hash>();
```
Comment (LauraXiulan) [inline src/Qowaiv/Hashing/Hash.cs:34]: Not sealed? Or was this the base class?

File: specs/Qowaiv.Specs/TestTools/Svo.cs [hunk 2]
```diff
@@ -58,9 +58,9 @@ public static class Svo
     /// <summary>-69/17</summary>
     public static readonly Fraction Fraction = -69.DividedBy(17);
 
-    [Obsolete("Will be dropped in version 7. Use Qowaiv.Sex instead.")]
-    public static readonly Gender Gender = Gender.Female;
+    /// <summary>8a1a8c42-d2ff-e254-e26e-b6abcbf19420</summary>
```
[the remark this answers is not in the document]
Comment (Corniel) [reply specs/Qowaiv.Specs/TestTools/Svo.cs:61]: It does, as the mouse over shows the value.

File: src/Qowaiv.Data.SqlClient/Generated/Timestamp.generated.cs [hunk 3]
```diff
@@ -163,7 +163,7 @@ void IXmlSerializable.WriteXml(XmlWriter writer)
 }
 
 public partial struct Timestamp
-#if NET8_0_OR_GREATER
+#if NET7_0_OR_GREATER
```
[the remark this answers is not in the document]
Comment (Corniel) [reply src/Qowaiv.Data.SqlClient/Generated/Timestamp.generated.cs:166]: Oops.

File: .editorconfig [hunk 4]
```diff
@@ -105,7 +105,7 @@ dotnet_diagnostic.S3376.severity = none # EventArgs, and Exception type names sh
 dotnet_diagnostic.S3376.severity = none # Attribute, EventArgs, and Exception type names should end with the type being extended
 dotnet_diagnostic.S4136.severity = none # Method overloads should be grouped together
 
-dotnet_diagnostic.S1133.severity = suggestion # Deprecated code should be removed
+# dotnet_diagnostic.S1133.severity = suggestion # Deprecated code should be removed
```
[the remark this answers is not in the document]
Comment (Corniel) [reply .editorconfig]: It was upgraded to a warning. But nicely spotted.

**Review by LauraXiulan — APPROVED** [review]

**Pull request merged**